bcdaworker/worker/worker.go

```diff
@@ -70,28 +72,28 @@ func (w *worker) ProcessJob(ctx context.Context, job models.Job, jobArgs models.
 
 	aco, err := w.r.GetACOByUUID(ctx, job.ACOID)
 	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("ProcessJob: could not retrieve ACO from database by UUID %s", job.ACOID))
+		err = errors.Wrap(err, fmt.Sprintf("ProcessJob: could not retrieve ACO from database by UUID %s", job.ACOID))
+		log.Worker.WithFields(logFields).Error(err)
+		return err
 	}
 
-	log.Worker.WithFields(logrus.Fields{
-		"jobID": jobArgs.ID,
-		"cmsID": *aco.CMSID,
-	})
+	logFields["cms_id"] = aco.CMSID
+	ctx = workerlog.WithLogFields(ctx, logFields)
```

Comment:
adding new fields to the context.

Reply:
So will each worker have its own personalized set of log fields depending on the contents of ctx? Would it be possible to document somewhere all of the different types of log fields that could be returned?

Reply:
the log fields won't be specific to the job but their values will be. The `logrus.Fields{}` will contain a new map of log values for each context that is created. Every time the worker processes a job, it creates a new context and when it is complete, the context is no longer used and the cycle starts all over again. 

The log fields are not documented anywhere but the code base and in a future PR i'd like to at least make the log field **names** be constants so we can avoid typos and that sort of thing.